### PR TITLE
add red ranges (both defensive and offensive ones)

### PR DESCRIPTION
**Pull request by stocchi**
both need some love from someone who know how to use route1
i added some _TODO_ with suggestions

### CODE REVIEW

**Comment by sizzle-psr** [comment]
Make sure to squash the commits whenever this is merged. I'll wait for tikevin's :+1: in order to merge.

File: resources/red/ranges/defensive ranges.md [hunk 1]
```diff
@@ -0,0 +1,771 @@
...
+   - Note : there are no [`rolls`] for stat changes because critical hits ignore all of them
+ - All rolls include 1/39 max roll
+
+## Squirtle
+
+**Rival** _TODO_: this is not accurate, needs to be fixed
+
+Bulbasaur    | Tackle
+------------ | -------------
+&nbsp;       | 2-3 [3-5]
```
Comment (sizzle-psr) [inline resources/red/ranges/defensive ranges.md:16]: Github doesn't seem to recognise this symbol in the diff. Is this intended behaviour?

**Review by sizzle-psr — CHANGES_REQUESTED** [review]
Please rename "defensive ranges.md" to "defensive-ranges.md" and the same for the offensive ranges file. Having spaces in file names is usually a recipe for trouble.

**Comment by stocchi** [comment]
fixing everything

File: resources/red/ranges/defensive ranges.md (hunk 1 above)
Comment (stocchi) [reply resources/red/ranges/defensive ranges.md:16]: i think so, its displayed only in the source

Comment (stocchi) [reply resources/red/ranges/defensive ranges.md:16]: yeah its a non-breaking-space

**Pull request closed without merging**